tests/scripts/all.sh

```diff
@@ -1853,6 +1853,9 @@ component_test_psa_crypto_config_accel_hash_use_psa () {
     scripts/config.py set MBEDTLS_PSA_CRYPTO_DRIVERS
     scripts/config.py set MBEDTLS_PSA_CRYPTO_CONFIG
     scripts/config.py set MBEDTLS_USE_PSA_CRYPTO
+    scripts/config.py set MBEDTLS_SSL_PROTO_TLS1_3
+    scripts/config.py set MBEDTLS_SSL_TLS1_3_COMPATIBILITY_MODE
+    scripts/config.py set MBEDTLS_SSL_CID_TLS1_3_PADDING_GRANULARITY 1
```

Comment:
I would prefer this to be placed at the end (where there is currently a comment about TLS 1.3, which should be changed btw) - first focus on the "driver-only hashes" aspect, then other config changes.